test/integration/framework/utils.go

```diff
@@ -368,7 +368,7 @@ func ScaleDeployment(setupContextTimeout time.Duration, client client.Client, de
 	}
 
 	// wait until scaled
-	if err := WaitUntilDeploymentScaled(ctxSetup, client, namespace, name, *desiredReplicas); err != nil {
+	if err := common.WaitUntilDeploymentScaledToDesiredReplicas(ctxSetup, client, namespace, name, *desiredReplicas); err != nil {
```

Comment:
This is the old testframework which will be soon removed.
So no need to update the function.